spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/pubsub/GcpPubSubProperties.java

```diff
@@ -32,11 +32,15 @@
 @ConfigurationProperties("spring.cloud.gcp.pubsub")
 public class GcpPubSubProperties implements CredentialsSupplier {
 
+	// Contains settings specific to the subscriber factory
 	private final Subscriber subscriber = new Subscriber();
 
+	// Contains settings specific to the publisher factory
 	private final Publisher publisher = new Publisher();
 
-	/** Overrides the GCP project ID specified in the Core module. */
+	/**
+	 * Overrides the GCP project ID specified in the Core module.
```

Comment:
Why haven't you fixed all similar issues in this class?
I understand that it's not your PR responsibility, but that is just simple refactoring...
Unfortunately I don't have time to walk through the project for such a fix, so I comment for flaws whenever I see them.

Thanks for understanding.

Reply:
I fixed the other one in this file that my eye didn't catch the first time.